packages/xo-server-test/package.json

```diff
@@ -36,13 +36,14 @@
     "golike-defer": "^0.4.1",
     "jest": "^24.8.0",
     "lodash": "^4.17.11",
+    "promise-toolbox": "^0.13.0",
     "xo-collection": "^0.4.1",
     "xo-common": "^0.2.0",
     "xo-lib": "^0.9.0"
   },
   "scripts": {
-    "dev-test": "jest --bail --watch",
-    "test": "jest"
+    "dev-test": "jest --bail --watch --runInBand",
```

Comment:
I did not really understand what you mean. You think I should just add a comment in the `README.md` to prevent running tests with the `--runInBand` option ?